keda/templates/22-metrics-deployment.yaml

```diff
@@ -86,10 +86,10 @@ spec:
           env:
             - name: WATCH_NAMESPACE
               value: {{ .Values.watchNamespace | quote }}
-            {{- if .Values.http.timeout }}
             - name: KEDA_HTTP_DEFAULT_TIMEOUT
               value: {{ .Values.http.timeout | quote }}
-            {{- end }}
+            - name: KEDA_HTTP_MIN_TLS_VERSION
```

Comment:
What about our interceptor/webhooks?

Reply:
Webhooks don't have any HttpClient (created by us). I was thinking about supporting thisor not in the metrics server as the metric querying is deprecated there, and I added it for security reasons more than for giving options

Reply:
Ok; fair enough